resources/kiali/values.yaml

```
@@ -323,7 +332,8 @@ kiali:
     # a single, specific Kiali version, thus this setting may have no effect depending on how the
     # operator itself was configured.
     #    ---
-      image_version: "1.38.1-e0da5a68"
+    #.   ATTENTION: this setting is will not be effective as it got replaced by the global.images.kiali setting. 
```

```suggestion
    #.   ATTENTION: This setting will not be effective. It has been replaced with the global.images.kiali setting. 
```

Again, consider putting this right after line 321.